problems/13.Merge Sorted Array.gs

```diff
@@ -0,0 +1,35 @@
+/**
+ Do not return anything, modify nums1 in-place instead.
+ */
+function merge(nums1: number[], m: number, nums2: number[], n: number): void {
+
+    for (let i = 0; i < n; i++) {
+        nums1[m + i] = nums2[i]
+    }
+
+
+    sortiraj(nums1)
+
+};
+
+function sortiraj(niz: number[]): number[] {
```

Comment:
You should do the follow up:
"Follow up: Can you come up with an algorithm that runs in O(m + n) time?"
What you have done is just merged array and sorted them in O(n^2). You haven't used the fact that **THEY ARE ALREADY SORTED**

You are doing to much work for nothing...